s3_file_field/widgets.py

```diff
@@ -39,23 +39,16 @@ class S3FileInput(ClearableFileInput):
     class Media:
         js = ['joist/joist.js']
 
-    baseurl: Optional[str] = settings.S3FF_API_BASE_URL
-
     @property
     def template_name(self):
         if settings._S3FF_STORAGE_PROVIDER == StorageProvider.UNSUPPORTED:
             return 'django/forms/widgets/file.html'
         else:
             return 'joist/s3fileinput.html'
 
-    def __init__(self, attrs=None):
```

Comment:
Note, this removes a feature where `S3FileInput` widgets could override the `baseurl`, but I'm not sure how this would ever be used now, so I think it's fine to eliminate.